bindings/java/hyperic_jni/src/org/hyperic/jni/ArchNameTask.java

```diff
@@ -92,19 +92,19 @@ else if (ArchLoader.IS_AIX) {
             //default to most recent SDK
             //MacOSX10.3.9.sdk, MacOSX10.4u.sdk, MacOSX10.5.sdk,etc.
             File[] sdks =
-                new File("/Developer/SDKs").listFiles(new FileFilter() {
+                new File("/Library/Developer/CommandLineTools/SDKs").listFiles(new FileFilter() {
                     public boolean accept(File file) {
                         String name = file.getName();
                         return
-                            name.startsWith("MacOSX10.") &&
+                            name.startsWith("MacOSX1") &&
```

Comment:
Not sure if it is compatible for Intel macs. But you provide package worked for me.

Reply:
Yes it is, this is just to catch later versions of macOS.